app/src/main/java/com/codesoom/assignment/DemoHttpHandler.java

```diff
@@ -0,0 +1,107 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.ByteArrayOutputStream;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DemoHttpHandler implements HttpHandler {
+    private ObjectMapper objectMapper = new ObjectMapper();
+    private List<Task> tasks = new ArrayList<>();
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        String method = exchange.getRequestMethod();
+        URI uri = exchange.getRequestURI();
+        String path = uri.getPath();
+        InputStream inputStream = exchange.getRequestBody();
+
+        String body = new BufferedReader(new InputStreamReader(inputStream)).lines().collect(Collectors.joining("\n"));
+
+        System.out.println(method + "" + path);
+
+        if (!body.isBlank()) {
+            System.out.println(body);
+
+            Task task = toTask(body);
+            tasks.add(task);
+
+            System.out.println(task);
+        }
```

Comment:
우리가 글을 작성할 때 관련이 있는 것과 관련이 없는 것을 문단으로 구분하는 것처럼, 코드도 동일하게 적절하게 잘 묶어주면
코드를 읽는 사람이 더 잘 읽을 수 있습니다.

```suggestion
        InputStream inputStream = exchange.getRequestBody();
        String body = new BufferedReader(
            new InputStreamReader(inputStream)
        ).lines().collect(Collectors.joining("\n"));
        if (!body.isBlank()) {
            Task task = toTask(body);
            tasks.add(task);
        }
```